nrfx/hal/nrf_nvmc.h

```diff
@@ -35,6 +35,7 @@
 #define NRF_NVMC_H__
 
 #include <nrfx.h>
+#include <string.h>
```

Comment:
could be added to `nrfx_common.h` instead, under other standard header inclusions